Assets/HotmaxEditorVR/builder/editorModeUIController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class editorModeUIController : MonoBehaviour
```

Comment:
I guess I have questions with how functionality is split between this class and editorModeUI. It seems like there are some "controller"-esque behavior in the editorModeUI class for instance.

Reply:
Also I'm not sure about the naming classes "Controller", only because we aren't really following an MVC model, so people might get confused with the nomenclature.. I think "Manager" might be better. 
Same with PropController --> Prop(s)Manager.